server/midddleware/auth.go

```diff
@@ -100,7 +101,16 @@ func AuthFunction(ctx context.Context, jwtValidator *validator.Validator, config
 			}
 		}
 	}()
-
+	// disable health check authn/z
+	var fullMethodName string
+	fullMethodNameFound := false
+	clientCtx := inprocgrpc.ClientContext(ctx)
+	if clientCtx != nil {
+		fullMethodName, fullMethodNameFound = runtime.RPCMethod(clientCtx)
```

Comment:
also shouldn't this live in request_metadata? `GetMethodFullName(ctx) (string, bool)`

Reply:
yes you are right this is more suitable there. moved it to request_metadata.